frontend/src/components/History/EditableMetadataCell.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import { DebounceInput } from "react-debounce-input";
+
+export default function HistoryMetadataCell({ id, value, modifyMetadataValue }) {
```

Comment:
Would it be possible to add tooltips to the headers so when you hover on a column it will give the full title? The metadata field names get cut off at about 11 characters which could make it hard to distinguish them.

Reply:
Added for all metadata columns